corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -206,6 +206,52 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
               </div>
             {% endif %}
 
+            {% if request|request_has_privilege:"CLOUDCARE" %}
+            <legend>
+              {% trans "Web Apps" %}
+            </legend>
```

Comment:
Did this look like a decent target to extract into a template?  Not sure how much it has in common with the other parameterized permissions